extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -69,20 +71,20 @@ class AmpStickyAd extends AMP.BaseElement {
 
   /** @override */
   unlayoutCallback() {
+    this.viewport_.updatePaddingBottom(0);
     return true;
   }
 
   /** @override */
   detachedCallback() {
+    this.viewport_.updatePaddingBottom(0);
```

Comment:
I'm not sure this one should be necessary. Any mutations that would have wrapped it should already have happened, no?